dd-sdk-android/src/main/kotlin/com/datadog/android/Datadog.kt

```diff
@@ -210,6 +218,16 @@ object Datadog {
         userInfoProvider.setUserInfo(UserInfo(id, name, email))
     }
 
+    /**
+     * Creates a tracer builder instance.
+     * @param serviceName as the name of a set of processes that do the same job.
+     * Used for grouping stats for your application.
+     */
+    @JvmStatic
+    fun tracerBuilder(serviceName: String): TracerBuilder {
```

Comment:
Hmm, so we'll have:
* `Logger.builder.set(serviceName: "...").build()`

and

* `Datadog.tracerBuilder(serviceName: "...")..build()`

public API which is kinda asymmetric. Why not `Tracer.builder.set(serviceName: "...").build()`?

Reply:
Yeah...I know, I was just trying to find a way to not expose the AndroidTracerBuilder class and just the interface (TraceBuilder). But you're right maybe I should move that there.

Reply:
Solved